cli/errors.rs

```diff
@@ -1,15 +1,19 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
Should we rename this file to deno_error.rs ? 

(Originally I thought it would contain all the error handling code, but that's not the case anymore.)

Reply:
:+1: yeah, I think it was getting unwieldy to have everything in a single module.